cmd/build/helmify/main.go

```diff
@@ -137,7 +137,7 @@ func (ks *kindSet) Write() error {
 			}
 
 			if kind == DeploymentKind {
-				obj = strings.Replace(obj, "      labels:", "      labels:\n{{- include \"gatekeeper.podLabels\" . }}", 1)
+				obj = strings.Replace(obj, "      labels:", "      labels:\n{{- include \"gatekeeper.podLabels\" . | nindent 8 }}", 1)
```

Comment:
```suggestion
				obj = strings.Replace(obj, "      labels:", "      labels:\n        {{- include \"gatekeeper.podLabels\" . | nindent 8 }}", 1)
```

based on https://github.com/open-policy-agent/gatekeeper/issues/3145#issuecomment-1790622112 isn't this missing here to fix the intendation of the templating var?

Reply:
The indent is not needed as the function indents the whole block. You can test it on the template. 
I believe the space would be removed by the `-` at the beginning of the template block so it will not change anything.